docs/manual/forms/examples/dynamic_options.rst

```diff
@@ -0,0 +1,75 @@
+.. _example_logic_dynamic_options:
+
+Formulier met dynamische opties
+===============================
+
+In dit voorbeeld maken we een deel-formulier bestaande uit 1 stap, waarbij
+de opties van een "radio" component dynamisch worden aangepast afhankelijk van de ingevulde antwoorden van een ander
+veld.
+
+Deze functionaliteit kan op dezelfde manier worden gebruikt voor de "keuzelijst" component en de "selectievakjes"
+component.
+
+
+Formulier maken
+===============
+
+#. Maak een formulier aan met de volgende gegevens:
+
+    * **Naam**: Toeslag aanvraag demo
+
+#. Klik op het tabblad **Stappen en velden**.
+#. Klik aan de linkerkant op **Stap toevoegen** en selecteer **Maak een nieuwe
+   formulierdefinitie**.
+#. Onder de sectie **(Herbruikbare) stapgegevens** vul het volgende in:
+
+    * **Naam**: Persoonlijke gegevens
+
+#. Scroll naar de sectie **Velden**.
+#. Sleep een **Herhalende groep** component op het witte vlak, vul de volgende
+   gegevens in en druk daarna op **Opslaan**:
+
+   * **Label**: Adressen
+
+#. Sleep deze componenten binnen de herhalende groep:
+
+    #. **Postcode** component met label *Postcode*
+    #. **Tekstveld** component met label *Huisnummer en toevoeging*
+    #. **Tekstveld** component met label *Straatnaam*
+    #. **Tekstveld** component met label *Stad*
+
+#. Sleep een **radio** component op het witte vlak, vul de volgende
+   gegevens in en druk daarna op **Opslaan**:
+
+   * **Label**: Waar ontvangt u graag post?
+   * **Data bron**: *variabel*
```

Comment:
```suggestion
   * **Data bron**: *variabele*
```
Translation will need updating I presume?

Reply:
Is there some Sphinx plugin that can replace these references to literals in the UI with the actual literals used in the application? I took this as an adjective about `Data bron` which is clearly wrong.

Reply:
Don't think so, and even then, it's coming from JS code dealing with formio & those related translations... Easier to keep track of it manually via PR review.